recipes/crowcpp-crow/all/conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, tools, CMake
+import os
+
+class CrowConan(ConanFile):
+    name = "crowcpp-crow"
+    homepage = "https://github.com/CrowCpp/crow"
+    description = "Crow is a C++ microframework for running web services."
+    topics = ("conan", "web", "microframework", "header-only")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "arch", "build_type"
+    exports_sources = ["patches/*"]
+    license = "BSD-3-Clause"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("boost/1.75.0")
+        self.requires("openssl/1.1.1i")
```

Comment:
I think you can remove openssl from the requirements.
It's only needed when you're using ssl in your library (which you e.g. are not using in your test_package).
See https://github.com/CrowCpp/crow/blob/45c41b6de616f340e3057fd73d69095b90e50a60/examples/CMakeLists.txt#L13-L17

Reply:
you're right, when my PR in the project is going to be merged, openssl will not be required in conan anymore, Until the 0.2 version this is unfortunately necessary because of not-deactivable examples where openssl is used